crates/common/tedge_config/src/system_services/managers/config.rs

```diff
@@ -22,6 +24,23 @@ pub struct InitConfig {
     pub is_active: Vec<String>,
 }
 
+#[derive(Deserialize, Debug, Eq, PartialEq)]
+#[serde(deny_unknown_fields)]
+pub struct SystemVariablesConfig {
+    pub restart: Vec<String>,
```

Comment:
I would prefer to use the term `reboot` (the device), since the verb `restart` (a service) is already used by `InitConfig`.